.gitignore

```diff
@@ -119,6 +119,7 @@ kubernetes.tar.gz
 !\.drone\.sec
 
 /bazel-*
+BUILD.bazel
```

Comment:
we should probably not be ignoring this

Reply:
It looks like my `bazel run //:gazelle` generated some new build files then (included in the next commit). Sorry about that, I'm a bazel noob.

Reply:
these build files do appear to be missing currently, probably because this repo has no CI ironically.